src/test/java/cc/protea/spreedly/SpreedlyTest.java

```diff
@@ -28,6 +32,58 @@ public void testGetGatewayProviders() throws Exception {
     	Assert.assertTrue(! list.isEmpty());
     }
 
+	@Test
+	public void testCreate() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
+		SpreedlyGatewayAccount account = new SpreedlyGatewayAccount();
+		account.gatewayType = "sage";
+		SpreedlyGatewayCredential c = new SpreedlyGatewayCredential();
+		c.name = "merchant_id_number";
+		c.value = "12345";
+		account.credentials.add(c);
+
+		final SpreedlyGatewayAccount createdAccount = spreedly.create(account);
+		Assert.assertNotNull(createdAccount);
+		Assert.assertTrue(createdAccount.getToken() == "sage");
+		Assert.assertTrue(createdAccount.getGatewayType() == "sage");
+		Assert.assertTrue(!createdAccount.getCredentials().isEmpty());
+		Assert.assertTrue(createdAccount.getCredentials().get(0).getName() == "merchant_id_number");
+		Assert.assertTrue(createdAccount.getCredentials().get(0).getValue() == "12345");
+
+		testAccount = createdAccount;
+	}
+
+	@Test
+	public void testRedact() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
+
+		SpreedlyGatewayAccount redactedAccount = spreedly.redact(testAccount);
+		Assert.assertTrue(redactedAccount.getState() == SpreedlyGatewayAccountState.REDACTED);
+	}
+
+	@Test
+	public void testRetain() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
+		SpreedlyGatewayAccount retainedAccount = spreedly.retain(testAccount);
+		Assert.assertTrue(retainedAccount.getState() == SpreedlyGatewayAccountState.RETAINED);
+	}
+
+	@Test
+	public void testGetGatewayAccounts() throws Exception {
+		if (spreedly == null) {
+			return;
+		}
+		final List<SpreedlyGatewayAccount> list = spreedly.listGatewayAccounts();
+		Assert.assertNotNull(list);
+		Assert.assertTrue(! list.isEmpty());
```

Comment:
Nit: name could be improved.
minimal change, name doesn't match call (GetGateway vs listGateway)
or name could be more honest and complete (testListGatewayAccountsReturnsAtLeastOneAccount)
even better, name the dynamic intent of the test (tell us the thing that can change during run time that is being tested here).